addons/artillerytables/functions/fnc_calculateMuzzleVelocity.sqf

```diff
@@ -20,7 +20,7 @@
 params ["_muzzleVelocity", "_temperature", "_airDensity"];
 
 // Calculate air density
-private _relativeDensity = _airDensity / 1.225;
+// private _relativeDensity = _airDensity / 1.225; // unused
```

Comment:
@LorenLuke - we can also drop the arg if this looks ok

Reply:
Imo we should nuke it instead of commenting it out.